### DISCUSSION TITLE
Sozi for geometry.

### DISCUSSION THREAD

**Discussion by amelillo** [senshu/Sozi#576] [Show and tell]
Hello everyone!
Here are some of the presentation I made to teach geometry and technical drawing. Ain't they nice? :-)

https://alexmelillo.altervista.org/tecnodispense/proiezioni_ortogonali.html

https://alexmelillo.altervista.org/tecnodispense/assonometria_isometrica.html

https://alexmelillo.altervista.org/tecnodispense/sviluppo_cubo.html

https://alexmelillo.altervista.org/tecnodispense/parallele.html

https://alexmelillo.altervista.org/tecnodispense/stella_a_sei_punte.html

**Comment by aumouvantsillage** [comment]
> Ain't they nice? :-)

Yes they are!

I was thinking that some of these examples would benefit from a 3D version of Sozi, where layers would be allowed to rotate along any axis.
